File: app/src/main/java/org/sco/movieratings/api/response/MoviesResponse.java
package org.sco.movieratings.api.response;

import java.io.Serializable;
import java.util.List;

import org.sco.movieratings.api.models.Movie;

import com.google.gson.annotations.SerializedName;

public class MoviesResponse implements Serializable {

    private int page;

    @SerializedName("results")
    private List<Movie> movies;

    @SerializedName("total_results")
    private int totalMovieCount;

    @SerializedName("total_pages")
    private int totalPageCount;

    public List<Movie> getMovies() {
        return movies;
    }

}

File: app/src/main/java/org/sco/movieratings/api/response/MoviesResponse.kt
package org.sco.movieratings.api.response

import com.google.gson.annotations.SerializedName
import org.sco.movieratings.api.models.Movie

data class MoviesResponse(
    @SerializedName("results") val movies: List<Movie>,
    @SerializedName("total_results") val totalMovieCount: Int,
    @SerializedName("total_pages") val totalPageCount: Int) {}